jarbas/chamber_of_deputies/views.py

```diff
@@ -77,25 +44,37 @@ def _build_filters(self):
         if in_latest is not None:
             self.queryset = self.queryset.in_latest_dataset(in_latest)
 
+        # filter search_vector
+        vector_param = self._vector_param()
+        if vector_param:
+            self.queryset = self.queryset.search_vector(vector_param)
+
         # filter queryset
         if filters:
             self.queryset = self.queryset.tuple_filter(**filters)
 
-    def _build_vector_filters(self):
-        """ 
-            Builder and aplly all vector filters on QuerySet 
-        """
+        # change ordering if needed
+        order_by = self.request.query_params.get('order_by')
+        if order_by == 'probability':
+            self.queryset = self.queryset.order_by_probability()
 
-        # get vector parameters from query string
-        vector_params = (
-            'congressperson_name',
-        )
-        vector_values = map(self.request.query_params.get, vector_params)
-        vector_filters = {k: v for k, v in zip(vector_params, vector_values) if v}
+        return super().get(request)
 
-        # filter search_vector
-        for vector in vector_filters:
-            self.queryset = self.queryset.annotate(search=SearchVector(vector)).filter(search=vector_filters[vector])
+    def _bool_param(self, param):
+        if param not in self.request.query_params:
+            return None
+
+        value = self.request.query_params[param]
+        if value.lower() in ('1', 'true'):
+            return True
+
+        return False
```

Comment:
We could try to delete lines 64-66 by replacing `value = self.request.query_params[param]` by `value = self.request.query_params.get(param)` I guess. This method could look like:

```python
    def _bool_param(self, param):
        value = self.request.query_params.get(param)
        return value.lower() in ('1', 'true')
```